dotnet/samples/KernelSyntaxExamples/Example12_SequentialPlanner.cs

```diff
@@ -221,6 +256,31 @@ private static async Task MemorySampleAsync()
         Console.WriteLine(plan.ToPlanWithGoalString());
     }
 
+    private static IKernel InitializeKernelWithMemory()
+    {
+        // IMPORTANT: Register an embedding generation service and a memory store. The Planner will
+        // use these to generate and store embeddings for the function descriptions.
+        var kernel = new KernelBuilder()
+            .WithLogger(ConsoleLogger.Logger)
+            .WithAzureChatCompletionService(
+                TestConfiguration.AzureOpenAI.ChatDeploymentName,
+                TestConfiguration.AzureOpenAI.Endpoint,
+                TestConfiguration.AzureOpenAI.ApiKey)
+            .WithAzureTextEmbeddingGenerationService(
+                TestConfiguration.AzureOpenAIEmbeddings.DeploymentName,
+                TestConfiguration.AzureOpenAIEmbeddings.Endpoint,
+                TestConfiguration.AzureOpenAIEmbeddings.ApiKey)
+            .WithMemoryStorage(new VolatileMemoryStore())
+            .Build();
+
+        return kernel;
+    }
+
+    private static ISemanticTextMemory GetMemory()
```

Comment:
Creating a Kernel to get the .Memory property seems as an overkill.

Would remove this method